cli/src/cli-cmd.c

```diff
@@ -28,7 +28,7 @@ static pthread_mutex_t conn_mutex = PTHREAD_MUTEX_INITIALIZER;
 int cli_op_ret = 0;
 static gf_boolean_t connected = _gf_false;
 
-static unsigned
+static int64_t
```

Comment:
Maybe using `time_t` would be better here to make it clear that it returns a time, not a boolean as the function name could imply.

Reply:
Got it. Thanks.